onnxruntime/test/onnx/TestResultStat.h

```diff
@@ -75,5 +75,5 @@ class TestResultStat {
   onnxruntime::OrtMutex m_;
   std::unordered_set<std::string> not_implemented_kernels;
   std::unordered_set<std::string> failed_kernels;
-  std::unordered_set<std::string> failed_test_cases;
+  std::vector< std::pair<std::string, int64_t> > failed_test_cases;
```

Comment:
>std::vector< [](start = 2, length = 12)

 a set allows you to quickly query the entry. Why the vector?

Reply:
Since we only need to go over "failed_test_cases", no query required.